build/bind/dappnode.io.hosts

```diff
@@ -0,0 +1,9 @@
+$TTL    3600
+@       IN      SOA     dyndns.dappnode.io. root.dyndns.dappnode.io.(
+                   2007010401           ; Serial
+                         3600           ; Refresh [1h]
+                          600           ; Retry   [10m]
+                        86400           ; Expire  [1d]
+                          600 )         ; Negative Cache TTL [1h]
+;
+@       IN      NS      dyndns.dappnode.io.
```

Comment:
Is it possible to resolve other external dappnodes using this zone?

Reply:
Yes, since this zone is going to be the fullname:
```
/usr/src/app # rndc showzone eXXXXXX2360687f3.dyndns.dappnode.io
zone "eXXXXXX2360687f3.dyndns.dappnode.io" in internal_domain { type master; file "/etc/bind/dappnode.io.hosts"; };
```
So it only applies for that domain